File: 1442. Count Triplets That Can Form Two Arrays of Equal XOR.go
func countTriplets(arr []int) int {
	answer := 0
	for i := 0; i < len(arr); i++ {
		xor := arr[i]
		for j := i + 1; j < len(arr); j++ {
			xor ^= arr[j]
			if xor == 0 {
				answer += j - i
			}
		}
	}
	return answer
}

File: 1442. Count Triplets That Can Form Two Arrays of Equal XOR.py
class Solution:
    def countTriplets(self, arr: List[int]) -> int:
        answer = 0
        for i in range(0, len(arr)):
            xor == ar[i]
            for j in range(i+1, len(arr)):
                xor ^= arr[j]
                if xor == 0: answer += j-i
        return answer